pkgs/development/python-modules/orange3/default.nix

```diff
@@ -0,0 +1,122 @@
+{ lib
+, buildPythonPackage
+, copyDesktopItems
+, fetchurl
+, makeDesktopItem
+, fetchFromGitHub
+, nix-update-script
+, pythonRelaxDepsHook
+, baycomp
+, bottleneck
+, chardet
+, cython
+, httpx
+, joblib
+, keyring
+, keyrings-alt
+, matplotlib
+, numpy
+, openpyxl
+, opentsne
+, orange-canvas-core
+, orange-widget-base
+, pandas
+, pyqtgraph
+, pyqtwebengine
+, python-louvain
+, pyyaml
+, qt5
+, qtconsole
+, requests
+, scikit-learn
+, scipy
+, serverfiles
+, xlrd
+, xlsxwriter
+}:
+
+buildPythonPackage rec {
+  pname = "Orange3";
+  version = "3.35.0";
+
+  src = fetchFromGitHub {
+    owner = "biolab";
+    repo = "orange3";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-dj/Z4uOjA4nidd45dnHZDyHZP6Fy/MGC8asqOPV7U7A=";
+  };
+
+  pythonRelaxDeps = [
+    "scikit-learn"
+    "pyqtgraph" # TODO: remove after bump with a version greater than 0.13.1
+  ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    cython
+    pythonRelaxDepsHook
+    qt5.wrapQtAppsHook
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    chardet
+    openpyxl
+    opentsne
+    qtconsole
+    bottleneck
+    matplotlib
+    joblib
+    requests
+    keyring
+    scikit-learn
+    pandas
+    pyqtwebengine
+    serverfiles
+    orange-canvas-core
+    python-louvain
+    xlrd
+    xlsxwriter
+    httpx
+    pyqtgraph
+    orange-widget-base
+    keyrings-alt
+    pyyaml
+    baycomp
+  ];
+
+  doCheck = false; # TODO: xvfb stuff
+
+  desktopItems = [
+    (makeDesktopItem { # https://github.com/biolab/orange3/blob/master/distribute/orange-canvas.desktop
+      name = "orange";
+      exec = "orange-canvas";
+      desktopName = "Orange Data Mining";
+      genericName = "Data Mining Suite";
+      comment = "Explore, analyze, and visualize your data";
+      icon = "orange-canvas";
+      mimeTypes = [ "application/x-extension-ows" ];
+      categories = [ "Science" "Education" "ArtificialIntelligence" "DataVisualization" "NumericalAnalysis" "Qt" ];
+      keywords = [ "Machine Learning" "Scientific Visualization" "Statistical Analysis" ];
+    })
```

Comment:
Instead of replicating the file here can't you just copy it to the output?

Reply:
No idea

I am kinda following the rfc42 mindset xD